a12.c

```diff
@@ -0,0 +1,38 @@
+
+#include <stdio.h>
+
+int fibonacci(int n) {
+    int t1 = 0, t2 = 1, nextTerm;
+
+    for (int i = 1; i <= n; ++i) {
+        printf("%d, ", t1);
+        nextTerm = t1 + t2;
+        t1 = t2;
+        t2 = nextTerm;
+    }
+    return 0;
+}
+
+int main() {
+    int n = 10; // number of terms to be printed
+    fibonacci(n);
+    return 0;
+}
+int isFibonacci(int n) {
+    if (n <= 0)
+        return 0;
+    int t1 = 0, t2 = 1, nextTerm = t1 + t2;
+    while (nextTerm < n) {
+        t1 = t2;
+        t2 = nextTerm;
+        nextTerm = t1 + t2;
+    }
+    return nextTerm == n;
+}
+        printf("%d is ", testNumbers[i]);
```

Comment:
Mi6: You made a mistake code from line 32 to 35 has to be inside main. Make sure the code is compilable

Reply:
Working on it...

*This is an automated message generated by Mi6*